ibis-server/app/model/mdl.py

```diff
@@ -0,0 +1,82 @@
+from enum import StrEnum, auto
+from typing import List, Optional, Dict
+
+from pydantic import BaseModel, Field
+
+
+class Manifest(BaseModel):
```

Comment:
I think we should change the dry-plan API in Wren Engine.
```json
{
   "manifest_json": "....",   // just json string
   "sql": "...",
   "modelingOnly": true
}
```

We don't need handle the DTOs everywhere.